detect_variants/add_bam_readcount_to_vcf.cwl

```diff
@@ -4,32 +4,28 @@ cwlVersion: v1.0
 class: CommandLineTool
 label: "add bam_readcount info to vcf"
 
-baseCommand: ["/usr/bin/python3", "/usr/bin/add_bam_readcount_to_vcf_helper.py"]
+baseCommand: ["vcf-readcount-annotator"]
+requirements:
+    - class: InlineJavascriptRequirement
```

Comment:
`InlineJavascriptRequirement` isn't necessary to use `$(runtime.outdir)` (as seen [here](https://github.com/genome/cancer-genomics-workflow/blob/00aa67b1fe0400cde023272dfa472bc56bae0314/qc/collect_insert_size_metrics.cwl#L8), for example).

Reply:
Argh, I always get that wrong. It's now removed.